src/frontend/src/lib/components/tokens/SkeletonTokenBalance.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
```

Comment:
For consistency, probably `TokenBalanceSkeleton` instead of `SkeletonTokenBalance`

Reply:
funny enough: i was undecided between the two and I tossed a coin eheh!

Changed